envoy/router/router.h

```diff
@@ -686,6 +686,15 @@ class VirtualHost {
    */
   virtual uint32_t retryShadowBufferLimit() const PURE;
```

Comment:
Is this the `per_request_buffer_limit_bytes` value? If so can we just rename this method to decrease confusion?